mangaki/mangaki/templates/base.html

```diff
@@ -86,9 +86,14 @@
                             </a>
                         </li>
                         <li class="dropdown">
-                            <a href="{% url 'top' 'directors' %}">
-                                {% trans "Top 20" %}
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+                                {% trans "Top 20" %} <span class="caret"></span>
                             </a>
+                            <ul class="dropdown-menu">
```

Comment:
Could you add back the a11y tags (aria-*** / role stuff) to not deteriorate the very low accessibility we currently have :D ?